src/production.js

```diff
@@ -14,13 +14,15 @@ const testnetNode = new TestnetNode({
     account: '0x' + 'fa5b9836c46b6559be750b2f3c12657081fab858'.toUpperCase(),
     provider: 'http://localhost:8545',
     contract: '0x8B3FBD781096B51E68448C6E5B53B240F663199F',
-    prefix: 'test.crypto.eth.rinkeby.'
+    prefix: 'test.crypto.eth.rinkeby.',
+    rate: Buffer.from([ 207, 65, 112, 139 ]) // 207 * (256*256*256) + 65 * (256*256) + 112 * (256) + 139 = 3477172363 = 3.4 * 10^9 Wei = 3.4 * 10^-9 ETH = 1 / 287590000 ETH = 1 micro USD
```

Comment:
`ilp-connector` [fetches rates automatically](https://github.com/interledgerjs/ilp-connector/blob/53f83a4aaf1c7a8deaf7f58c54e0aa9e3911bdf5/src/backends/fixerio-plus-coinmarketcap.js). Maybe you can reuse the code.